litmus-portal/frontend/src/views/Analytics/DataSources/Table/TableData.tsx

```diff
@@ -86,240 +103,177 @@ const TableData: React.FC<TableDataProps> = ({ data }) => {
           <LightPills variant="danger" label={data.health_status} />
         )}
       </StyledTableCell>
-      <StyledTableCell className={classes.dataSourceName}>
-        <Typography variant="body2">
-          <strong>{data.ds_name}</strong>
+      <StyledTableCell className={classes.columnDivider}>
+        <Typography
+          className={`${classes.tableObjects} ${classes.dataSourceNameColData}`}
+        >
+          {data.ds_name}
         </Typography>
       </StyledTableCell>
-      <StyledTableCell className={classes.dataSourceType}>
-        <Typography variant="body2">
-          <strong>{data.ds_type}</strong>
+      <StyledTableCell className={classes.dividerPadding}>
+        <Typography
+          className={classes.tableObjects}
+          style={{ maxWidth: '5rem' }}
```

Comment:
Avoid inline styling at all instancex

Reply:
Yes but we have agreed to avoid duplicating styles using one single property override over existing class. PTAL at other PRs for reference. @whitetiger1399

Reply:
cc: @amityt @S-ayanide @Jonsy13

Reply:
cc: @arkajyotiMukherjee

Reply:
Is the class tableObject being used by other elements?

Reply:
yes @arkajyotiMukherjee only width changes